hooks/sentry/sentry_test.go

```diff
@@ -61,9 +61,12 @@ func TestSpecialFields(t *testing.T) {
 			t.Fatal(err.Error())
 		}
 		logger.Hooks.Add(hook)
+
+		req, _ := http.NewRequest("GET", "url", nil)
 		logger.WithFields(logrus.Fields{
-			"server_name": server_name,
-			"logger":      logger_name,
+			"server_name":  server_name,
+			"logger":       logger_name,
+			"http_request": req,
 		}).Error(message)
```

Comment:
`http_request` is added in this test, but not asserted in the below part.
Its attribute is `sentry.interfaces.Http` in the json data, not like other flat fields.
So `raven.Packet{}` can not include it as `raven.Packet.Interface`, and the data is removed.